contracts/contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -195,4 +195,29 @@ namespace eosiosystem {
       }
    }
 
+   void system_contract::fee( const name& payer, const asset& quantity ) {
+      require_auth( payer );
+      auto amount = quantity.amount;
+      auto to_per_block_pay = amount * 0.2; //TODO: move 0.2 to constants section
+      auto to_per_vote_pay  = amount * 0.7; //TODO: move 0.7 to constants section
+      auto to_remme         = amount - (to_per_block_pay + to_per_vote_pay);
+      if( amount > 0 ) {
+         {
+            token::transfer_action transfer_act{ token_account, { {_self, active_permission} } };
+            if( to_remme > 0 ) {
+               transfer_act.send( _self, saving_account, asset(to_remme, core_symbol()), "remme fee" ); //TODO: change saving_account => remme_account
```

Comment:
That is fine to keep the "saving" account name. Please do not use the term "fee"

Reply:
@roman-tik  What term should we use instead of "fee"?